tests/roles/ceph_migrate/tasks/mon.yaml

```diff
@@ -63,8 +63,7 @@
 
     - name: Fail mgr if active in the current node
       become: true
-      ansible.builtin.command:
-        "{{ ceph_cli }} mgr fail {{ mgr.active_name }}"
+      ansible.builtin.include_tasks: fail_mgr.yaml
```

Comment:
don't we need to pass mgr.active_name to the fail play/tasks?

Reply:
we don't, it's enough to keep it in the `when` condition to avoid an unnecessary execution of this command